phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -3507,23 +3525,30 @@ private void invalidateServerMetadataCacheWithRetries(Admin admin,
                     // DDL operations. We also need to think of we need separate RPC handler
                     // threads for this?
                     ServerRpcController controller = new ServerRpcController();
+                    for (InvalidateServerMetadataCacheRequest invalidateCacheRequest:
```

Comment:
We are already iterating over the list of `InvalidateServerMetadataCacheRequests` in `getRequest` to create the proto request. Can we do the logging there? The list is very small so maybe it does not matter?

Reply:
> The list is very small so maybe it does not matter?

Thats what I was thinking too. Atmost there will be 2 entries.

Reply:
It might still get rid of some code and bytes to string conversion.

Reply:
But then it won't have the region server name. Having a regionserver name would be helpful.

Reply:
Yeah that makes sense. In that case, do you think we can override `toString` in `InvalidateServerMetadataCacheRequest` to make it cleaner here?

Reply:
Yes, in the process of making that change.